scriptlets/_put

```diff
@@ -1,22 +1,43 @@
 #!/usr/bin/env bash
 
-# Transfer a file to a remote device via SCP.
+# Transfer files to a remote device via SCP.
 #
-# Dependencies:
-#   This script depends on the `set_ssh_options` script to set SSH options.
-#   DEVICE_IP and DEVICE_USER must be set in the environment.
+# Description:
 #
-# Usage:
-#   `_put <local_file> <remote_file>`
-#   Or source it in another script and call
-#   `put_function <local_file> <remote_file>`
+# The IP of the device is provided by the environment variable DEVICE_IP,
+# which must be set. The user name on the remote device is `ubuntu`
+# by default, unless provided by the environment variable DEVICE_USER.
+#
+# The last argument provided is considered to be the "target", i.e. where
+# on the remote device the file(s) will be copied. If the last argument is
+# `--`, then the file(s) will be copied on the home directory of DEVICE_USER
```

Comment:
why `--` to indicate /home/ubuntu? Does this help in cases where we don't necessarily know what the default user would be? I know that some devices use "admin" or "testuser" for example, but I think we will need the default user somehow for those though, don't we?
I only ask because this is a little opaque, and not clear where it's getting copied to when I saw it in use.

Reply:
I like the idea of using `:` instead. That seems a little more intuitive

Reply:
@plars `_put` now uses the colon to indicate the target and a single colon means the home directory of `DEVICE_USER`. This mirrors how `scp` and similar programs behave.